doc/maintainer/manual-build.rst

```diff
@@ -124,6 +124,9 @@ order, as it doesn't depend on other components::
 Linux toolchain
 ---------------
 
+uClibc toolchain
+````````````````
```

Comment:
http://www.sphinx-doc.org/en/stable/rest.html#sections

According to convention this should used `^^^^^`.